allauth_2fa/views.py

```diff
@@ -81,10 +82,14 @@ def _new_device(self):
         if not device_set.exists() or reset_device(self.request.user.pk):
             # reset device
```

Comment:
Above in line 69 user_has_valid_totp_device we are checking if user already has totp device setup but not email device setup